src/tests/ftest/pool/create_all_base.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+"""
+(C) Copyright 2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from apricot import TestWithServers
+
+
+class PoolCreateAllBaseTests(TestWithServers):
```

Comment:
```suggestion
class PoolCreateAllTestBase(TestWithServers):
```
Also, we usually place the test base file in ftest/util, so it would better if you could move this file there.